src/utils/addons/diffs/options.js

```diff
@@ -1,9 +1,7 @@
 const ansiStyles = require('ansi-styles')
 const chalk = require('chalk')
 
-const forceColor = new chalk.constructor({
-  enabled: true,
-})
+const forceColor = new chalk.Instance({ level: 1 })
```

Comment:
`chalk.constructor` was removed in favor of `chalk.Instance` via https://togithub.com/chalk/chalk/issues/322
Also the previous code assumed `enabled: true` forces color, which is not always the case:
https://github.com/chalk/chalk/pull/356/files#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15L123

That's one of the reasons `enabled` was removed in favor of just `level`